src/cdk/v2/destinations/yandex_metrica_offline_events/procWorkflow.yaml

```diff
@@ -0,0 +1,35 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - path: ../../bindings/jsontemplate
+    exportAll: true
+  - path: ./config
+  - name: removeUndefinedAndNullValues
+    path: ../../../../v0/util
+  - name: defaultRequestConfig
+    path: ../../../../v0/util
+  - path: ./utils
+
+steps:
+  - name: validateInput
+    template: |
+      let messageType = .message.type;
+      $.assert(messageType, "message Type is not present. Aborting message.");
+      $.assert(.message.type.toLowerCase() ==='identify', "Event type " + .message.type.toLowerCase() + " is not supported. Aborting message.");
+      $.assert(.message.traits || .message.properties, "Message traits/properties not present. Aborting message.");
+
+  - name: prepareData
+    template: |
+      let data = .message.traits
+      data = $.validateData(data)
+      let identifierType = .message.context.externalId[0].identifierType;
```

Comment:
From the source-code, it seems like `externalId` is an absolute need. Will we have it for all or should we have a validation for `externalId` as well ?

Reply:
We will have it from sources, but have added the test case based on existing validation